sdk/identity-v2/ci.yml

```diff
@@ -0,0 +1,77 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+
+trigger:
+  branches:
+    include:
+      - main
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+      - sdk/identity-v2/ci.yml
+      - sdk/identity-v2/azure-identity/
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+      - sdk/identity-v2/ci.yml
+      - sdk/identity-v2/azure-identity/
+
+parameters:
+  - name: release_dependsonlivetests
+    displayName: 'Release depends on live tests'
+    type: boolean
+    default: true
+  - name: release_azureidentity
+    displayName: 'azure-identity'
```

Comment:
Should we include `v2` in the display name? `azure-identity-v2` or `com.azure.v2:azure-identity`?